### PR TITLE
Update Foreman 3.6.2 Release Notes

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/foreman-3.6.2.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* hosts names are hidden in small screens  - https://projects.theforeman.org/issues/36263[#36263]
+* Host Detail button landed to old Host UI page - https://projects.theforeman.org/issues/36225[#36225]
+
+=== Packaging
+
+* Pin sass version to 1.60.z to avoid node 14 dependency - https://projects.theforeman.org/issues/36305[#36305]
+
+=== Unattended installations
+
+* include installation parameter to support puppet 8 - https://projects.theforeman.org/issues/36360[#36360]
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman-3.6.2.adoc:25]: This is an odd one: the issue is still in `New` so the [issues script](https://github.com/theforeman/theforeman-rel-eng/blob/master/issues) should have flagged this as present in Redmine, but not in git. I've now removed the Fixed in releases so it shouldn't show up anymore.

**Review by ekohl — CHANGES_REQUESTED** [review]
Perhaps some other advice is that I use the triaged flag to review. So https://projects.theforeman.org/projects/foreman/issues?utf8=%E2%9C%93&set_filter=1&sort=id%3Adesc%2C%2C&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1703&f%5B%5D=cf_5&op%5Bcf_5%5D=%3D&v%5Bcf_5%5D%5B%5D=0&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by= is a long URL, but it searches for Fixed in releases is 3.6.2 and Triaged is No.

Then I open them all and verify the category is correct.

File: guides/doc-Release_Notes/topics/foreman-3.6.2.adoc (hunk 1 above)
Comment (Griffin-Sullivan) [reply guides/doc-Release_Notes/topics/foreman-3.6.2.adoc:25]: So this should be removed?

File: guides/doc-Release_Notes/topics/foreman-3.6.2.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
+= Foreman 3.6.2
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1703[Redmine]
+
+== Foreman
+
+* Pin minitest &lt; 5.19 to resolve test failures - https://projects.theforeman.org/issues/36617[#36617]
```
[the remark this answers is not in the document]
Comment (Griffin-Sullivan) [reply guides/doc-Release_Notes/topics/foreman-3.6.2.adoc:7]: Do I need to change anything for this then?

**Comment by Griffin-Sullivan** [comment]
When I open that URL it doesn't return anything in the table. I'm not exactly sure what you're checking for since I can't see it.

**Comment by ekohl** [comment]
> When I open that URL it doesn't return anything in the table. I'm not exactly sure what you're checking for since I can't see it.

That's because I went through all issues and marked them as triaged.

File: guides/doc-Release_Notes/topics/foreman-3.6.2.adoc (hunk 1 above)
Comment (ekohl) [reply guides/doc-Release_Notes/topics/foreman-3.6.2.adoc:25]: If you regenerate the file it should be gone.

**Comment by Griffin-Sullivan** [comment]
> That's because I went through all issues and marked them as triaged.

Right, I understand that. But since I can't see the context, I don't know how this relates to my PR.

**Comment by ekohl** [comment]
> Right, I understand that. But since I can't see the context, I don't know how this relates to my PR.

It showed 3 Redmine issues: 1 was correct and I marked it as triaged, the other 2 I mentioned.

**Review by ekohl — APPROVED** [review]

**Pull request merged**